### PR TITLE
Code of Conduct  Update README.md

### PR DESCRIPTION
**Pull request by Rockyspade**
@Rockyspade #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Welcome to urfave/cli-docs/v3
+ Rockyspade/cli-docs # Welcome to urfave/cli-docs/v3
```
Comment (Juneezee) [inline README.md:1]: Huh?

**Comment by bartekpacia** [comment]
I think we can close this PR.

File: README.md (hunk 1 above)
Comment (Rockyspade) [reply README.md:1]: <Gh CLI/docs status merge <commit remote with to unfave/CLI-docs/v3 #7 #6 #5 #2

**Comment by abitrolly** [comment]
@bartekpacia I think we can report this account.

**Review by Rockyspade — COMMENTED** [review]
Readme.md

**Pull request closed without merging**